make/devices.cmake

```diff
@@ -0,0 +1,754 @@
+# This CMake include defines the following functions:
```

Comment:
Do you know if these comments can be automatically extracted and put into sphinx documentations in some way?

 - https://eb2.co/blog/2012/03/sphinx-and-cmake-beautiful-documentation-for-c---projects/
 - https://github.com/Kitware/CMake/tree/master/Utilities/Sphinx
 - https://cmake.org/pipermail/cmake/2017-June/065557.html

FYI I start trying to add sphinx integration here -> https://github.com/SymbiFlow/symbiflow-arch-defs/pull/44

Reply:
If I convert the comments to Sphinx comments it might work.  A task for a different pull I think.

Reply:
Yeap, can you add a github issue for it?